massgov/openmass
# DP-22960: Org subnavigation fixes for feature branch

**Description:**
Followup to fix the a link title bug in the "I want to" subnav menu item.

**Jira:** (Skip unless you are MA staff)
DP-22960


**To Test:**
- Go to /orgs/department-of-public-health
- Click on the "I want to..." menu item in the subnav.
- Verify the first link under "Find a service or task" is text and not a path.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
